### ISSUE TITLE
Take a look at mutation test issues

### ISSUE THREAD

**Issue by malukenho** [codelicia/xulieta#107]
It is polluting the diff making PR hard to read.

**Comment by EHER** [comment]
It is nice to have it but would make more sense to show it just on the changed files.

**Comment by malukenho** [comment]
I agree although I don't know how to do it because it is a feature on the GitHub side.

**Comment by malukenho** [comment]
I've fixed qodana erros here: https://github.com/codelicia/xulieta/pull/129. Now we only have infection cases 👍🏼

**Comment by malukenho** [comment]
We still need to add more tests to cover the non detected mutants
